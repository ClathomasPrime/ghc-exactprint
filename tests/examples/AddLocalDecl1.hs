module AddLocalDecl1 where

-- |This is a function
foo = x -- comment1

-- |Another fun
x = 3
