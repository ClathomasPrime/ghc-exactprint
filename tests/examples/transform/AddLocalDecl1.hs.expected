module AddLocalDecl1 where

-- |This is a function
foo = x -- comment1
  where
    nn = 2

-- |Another fun
x = a -- comment2
  where
    a = 3
